invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/deposit.less

```diff
@@ -129,3 +128,29 @@
 .item div.description:not(.license-item-description) {
   display: initial !important;
 }
+
+.deposits-community-header {
+  margin: -20px 0px 20px 0px;
```

Comment:
I am not a big fan of negative margins, because from my experience, it leads to problems with the rest of the layouts. Is there any other way to solve this ? Did you check `.top.attached` class of semantic ui containers?

Reply:
Using attached would attach it to the header wich is the one containing the bottom-margin and forcing all the rest to use negative margin. Instead I removed the margin from the header and also all the negative margin from other repos